### ISSUE TITLE
Notification buttons are inaccessible to screen readers

### ISSUE THREAD

**Issue by Neurrone** [Microsoft/vscode#50144]
Issue Type: <b>Bug</b>

1. Do anything that triggers a notification, e.g help -> check for updates. When an update is available, help -> install update.
2. At the bottom of the window, the screen reader shows the text of the notification about restarting Code to install updates. However, the buttons next to them are not visible to screen readers. I assume that extensions use the same API for notifications as well.

VS Code version: Code - Insiders 1.24.0-insider (912fff91ac4ff1bff9695e8da69abd729f7ae956, 2018-05-18T05:17:26.925Z)
OS version: Windows_NT x64 10.0.17134

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8550U CPU @ 1.80GHz (8 x 1992)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: disabled_software<br>video_decode: enabled<br>video_encode: enabled<br>vpx_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Memory (System)|15.86GB (10.55GB free)|
|Process Argv|C:\Program Files\Microsoft VS Code Insiders\Code - Insiders.exe|
|Screen Reader|yes|
|VM|40%|

</details><details><summary>Extensions (5)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-markdownlint|Dav|0.16.0
vscode-eslint|dba|1.4.9
python|ms-|2018.4.0
cpptools|ms-|0.17.1
markdown-preview-enhanced|shd|0.3.5

</details>

**Comment by Neurrone** [comment]
Just found the link in the status bar that moves focus to the notifications area; the link was only labelled with "2", so perhaps changing its label to "2 notifications" would make it easier to find, or perhaps adding a keyboard shortcut to instantly move focus to the notifications area.

**Comment by Neurrone** [comment]
@bpasero it depends on how many possible actions there are, but the way that normal windows notifications are announced do not include the list of possible actions to take, so just announcing the notification text should be fine.

For me the bigger issue is not being able to move keyboard focus to the buttons to see what actions are available and activate them.

**Comment by Neurrone** [comment]
Hm, what's in the image?

**Comment by alexandrudima** [comment]
I'll quickly jump in to describe the image.

The image shows a screenshot of the Keyboard Shortcuts editor (Ctrl+K Ctrl+S) and it searches in there for commands starting with "notification". Around 20 commands/keybindings are presented in the list.
